packages/mobile/ios/Podfile

```diff
@@ -25,6 +24,8 @@ target "celo" do
   pod "CeloBlockchain", :path => "../../../node_modules/@celo/client/CeloBlockchain.podspec", :modular_headers => true
 
   pod 'Segment-Firebase', '~> 2.7.0'
+  pod 'Segment-Adjust', '~> 3.1.1'
```

Comment:
Why are we touching Adjust in this PR?

Reply:
@etuleu  worked on this part so I'm not too sure, but we had a lot of dependency errors and had to play around with a lot of random dependency versions to resolve them

Reply:
Honestly I don't remember now why I needed this but I can double check if it is.

Reply:
I believe it is needed because I had to update

> segment/analytics-react-native

to 1.5.0

which means we need to also update the adjust lib